PRO-1054 Getting familiar (#50)
apostrophecms/a3-docs: docs/getting-started/README.md

::: tip
If you are doing active development on the project, you can also start the site with `npm run dev`. The app will then watch for changes, then restart the app and refresh the browser when it detects any.
:::

## Next step

> Now that Apostrophe is installed, go ahead and start building using the essentials [guide](/guide/). You can also continue getting to know Apostrophe in the [technical overview](/getting-started/technical-overview.md).

Author: Vuepress doesn't include that "Next -->" link to child pages from the section landing page, so I added this to lead to the technical overview. Maybe there's a better way?

::: tip
If you are starting the site in a production or do not want the process to watch for changes, start the site with `node app.js`.
:::

## Next steps

> Now that Apostrophe is installed, you're ready to start building. Check out the essentials [guide](/guide/) to get started or read more about Apostrophe's inner workings in the [technical overview](/getting-started/technical-overview.md).

Reviewer: inner workings still seems like the wrong phrase here. What are we really putting in this technical overview? Should we just...

```suggestion
Now that Apostrophe is installed, you're ready to start building. Check out the essentials [guide](/guide/) to get started or read the [technical overview](/getting-started/technical-overview.md) for an overview of Apostrophe's structure.
```

Author: Why is it not? It's colorful language, but [not slang](https://www.merriam-webster.com/dictionary/inner%20workings). And it's general enough to encompass both the directory structure and the tech stack stuff, the latter of which is not included in "Apostrophe's structure."
